doc/classes/Object.xml

```diff
@@ -406,8 +406,7 @@
 			<param index="0" name="method" type="StringName" />
 			<description>
 				Calls the [param method] on the object during idle time. Always returns null, [b]not[/b] the method's result.
-				Idle time happens mainly at the end of process and physics frames. In it, deferred calls will be run until there are none left, which means you can defer calls from other deferred calls and they'll still be run in the current idle time cycle. If not done carefully, this can result in infinite recursion without causing a stack overflow, which will hang the game similarly to an infinite loop.
-				This method supports a variable number of arguments, so parameters can be passed as a comma separated list.
```

Comment:
Not necessarily against this but I feel like `varargs` is always worth reminding as a grand majority of functions do.

Reply:
Oh my bad that wasn't intentional